content/docs/command-reference/remote/modify.md

```diff
@@ -271,6 +271,45 @@ including obtaining the necessary credentials, and how to form `gdrive://` URLs.
   $ dvc remote modify myremote gdrive_client_secret <client secret>
   ```
 
+- `gdrive_use_service_account` - instructs DVC to authenticate using a
+  [service account](/doc/user-guide/setup-google-drive-remote#configure-the-google-drive-remote-with-a-service-account).
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_use_service_account true
+  ```
+
+- `gdrive_service_account_email` - email address of the Google Project's service
+  account.
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_service_account_email <service acct email>
+  ```
+
+- `gdrive_service_account_p12_file_path` - Google Project's service account
+  `.p12` file path.
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_service_account_p12_file_path path/to/file.p12
+  ```
+
+- `gdrive_service_account_user_email` - email of the service account to
+  impersonate.
+
+  ```dvc
+  $ dvc remote modify myremote gdrive_service_account_user_email <user account email>
+  ```
+
+- `gdrive_trash_only` - configures DVC to move remote files to
+  [trash](https://developers.google.com/drive/api/v2/reference/files/trash)
+  instead of
+  [deleting](https://developers.google.com/drive/api/v2/reference/files/delete)
+  them permanently. `false` by default, meaning "delete". Useful for shared
+  drives, where delete permissions may not be given.
```

Comment:
Do we have a DVC Google org?

- [ ] Let's create a [shared drive](https://developers.google.com/drive/api/v2/about-shareddrives) to test all this with that kind of drive.

Reply:
Cc @shcheklein 🙂

Reply:
we do have DVC app, we do have Iterative org, you can share your personal drive with your work account to test stuff

Reply:
OMG I forgot about iterative.ai 🤦‍♂ The thing is though, if I create a folder on the org it will still be owned by me, and the Shared drives doc says:

> Shared drives support files owned by an organization rather than an individual user.

So maybe an org owner user has to create it? Or maybe it doesn't matter and its just all confusing wording...

Reply:
Yeah I created a folder in my iterative.ai GDrive but I'm pretty sure it's owned by me, not the org. I will share it with my personal Google acct and test around with it but that's a separate thing. Let's call it "shared folder".

If you can try to create an organization-owned shared drive and some folders and subfolders in it, that would be great @shcheklein. See https://support.google.com/a/users/answer/9310249?hl=en&ref_topic=9298590

Reply:
created the drive! resolving